config/i18n.js

```diff
@@ -1,5 +1,6 @@
 export default {
   langPickerRender: true,
-  langList: ['en-US', 'ru-RU'],
-  fallbackLng: 'en-US'
+  langList: ['en', 'ru'],
```

Comment:
Have you checked frontend after this change? I think you might have broken frontend. Please check that i18n works for frontend as well.

Reply:
The changes in semantics in configuration parameters should be supported by strong argumentation - why this is needed. Because you are forcing many projects based on this kit to swtich to update their configuration. If you don't have strong arguments, please return back original semantics - e.g. `en-US`, instead of `en`, `ru-RU`, instead of `ru`